website/docs/r/application_gateway.html.markdown

```diff
@@ -1,18 +1,18 @@
----
-layout: "azurerm"
-page_title: "Azure Resource Manager: azurerm_application_gateway"
-sidebar_current: "docs-azurerm-resource-application-gateway"
-description: |-
-  Manages an Application Gateway.
----
-
-# azurerm_application_gateway
-
-Manages an Application Gateway.
-
-## Example Usage
-
-```hcl
+---
```

Comment:
Did you reformat the whole markdown file? Looks like the commit introduces many unnecessary lines of changes.

Reply:
done